server/chat-plugins/scavenger-games.ts

```diff
@@ -451,6 +451,45 @@ const TWISTS: { [k: string]: Twist } = {
 		},
 	},
 
+	pointless: {
+		id: 'pointless',
+		name: 'Pointless',
+		desc: 'Players get bonus points for guessing the least commonly guessed answers.',
+		onAfterLoad() {
+			this.correct = this.questions.map(() => ({}));
+		},
+
+		onCorrectAnswer(player: ScavengerHuntPlayer, value: string) {
+			const curr = player.currentQuestion;
+
+			if (!this.correct[curr][value]) this.correct[curr][value] = [];
+			if (this.correct[curr][value].includes(player.id)) return;
+
+			this.correct[curr][value].push(player.id);
+		},
+
+		onAfterEnd(isReset) {
+			if (isReset) return;
+
+			const buffer = [];
+
+			for (const [idx, data] of this.correct.entries()) {
+				// collect the data for each question
+				const list = Object.entries<string[]>(data).map(([value, players]) => ({ players, count: players.length, value }));
+				const minValue = Math.min(...list.map(entry => entry.count));
+
+				const minEntries = list.filter(entry => entry.count === minValue);
+				const matchDisplay = minEntries.map(entry =>
+					this.questions[idx].answer.find(answer => toID(answer) === entry.value)).join(', ');
+				const playerDisplay = minEntries.flatMap(entry =>
+					entry.players.map(foundPlayer => this.players.find(player => player.id === foundPlayer)?.name)).join(', ');
```

Comment:
foundPlayer doesn't really need to be optional here (if it were, there'd also need to be a filter so the 'undefined' doesn't make it to the display) but not really an issue.